cmd/image.go

```diff
@@ -37,44 +49,28 @@ func printResultImg(results []Result) {
 	}
 }
 
-func checkImage(container apiv1.Container, image string, result *Result) {
-	var (
-		contImg string
-		contTag string
-	)
-
-	imageLabel := strings.Split(image, ":")
+func checkImage(container apiv1.Container, image imgFlags, result *Result) {
+	image.splitImageString()
+	contImage := imgFlags{img: container.Image}
+	contImage.splitImageString()
 
-	if len(imageLabel) < 2 {
-		log.Error("Image tag is missing!")
-		os.Exit(1)
-	}
-
-	compImg := imageLabel[0]
-	compTag := imageLabel[1]
-
-	contImgLabel := strings.Split(container.Image, ":")
-	contImg = contImgLabel[0]
-
-	if len(contImgLabel) < 2 {
-		if compImg == contImg {
+	if len(contImage.tag) == 0 {
+		if image.name == contImage.name {
 			// Image name was proper but image tag was missing
-			result.err = 1
+			result.err = ErrorImageTagMissing
 		}
 		return
 	}
 
-	contTag = contImgLabel[1]
-
-	if contImg == compImg && contTag != compTag {
+	if contImage.name == image.name && contImage.tag != image.tag {
 		result.err = 2
-		result.img_name = contImg
-		result.img_tag = contTag
+		result.img_name = contImage.name
```

Comment:
imgName `go` likes this format

Reply:
That's not my code! I mean thanks for catching that :)